Focus/Monomorphic.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  Monomorphic.swift
+//  Focus
+//
+//  Created by Robert Widmann on 8/23/15.
+//  Copyright © 2015 SypeLift. All rights reserved.
+//
+
+public struct MonomorphicLens<S, A> : LensType {
```

Comment:
How about `Simple(Lens|Iso|Prism)`?  That would be in line with the original `lens` library's conventions.